model/UsersData.ts

```diff
@@ -0,0 +1,151 @@
+import * as fsPromise from "fs";
+import { User } from "./User";
+import { File } from "./File";
+import { input } from "../utility/constants";
+
+export class UsersData {
+  public usersData: User[];
+  constructor() {
+    //Loading data from disk to in-memory
+    const data = fsPromise.readFileSync("./database/db.txt", "utf8");
+    let allUsers: any = data.split(",").slice(1);
+
+    allUsers = allUsers.map((element: any) => {
+      let bufferObj = Buffer.from(element, "base64");
+      return JSON.parse(bufferObj?.toString("utf8"));
+    });
+    this.usersData = allUsers;
+  }
+
+  public getUserData() {
```

Comment:
Need of getter for a public member userData?

Reply:
Resolved.